src/pages/Models/Details/_model-details.scss

```
@@ -1,4 +1,4 @@
-@import "../../scss/settings";
+@import "../../../scss/settings";
```

It seems node-sass was just silently failing here.